5.5/test/run

```diff
@@ -19,13 +19,18 @@ cleanup() {
 
 		echo "Stopping and removing container $CONTAINER..."
 		docker stop $CONTAINER
+		exit_status=$(docker inspect -f '{{.State.ExitCode}}' $CONTAINER)
+		if [ "$exit_status" != "0" ]; then
+			echo "Dumping logs for $CONTAINER"
+			docker logs $CONTAINER
+		fi
```

Comment:
ok, that was my lack of knowledge about trap.  if trap is still going to exit w/ the code that caused the trap to be invoked, that's fine.  just wanted to make sure we weren't eating the error.